django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,98 @@
+from django.core import checks
+from django.db import models
+from django.db.models import NOT_PROVIDED
+from django.db.models.expressions import DatabaseDefault
+from django.utils.translation import gettext_lazy as _
+
+__all__ = ("BigSerialField", "SmallSerialField", "SerialField")
+
+
+class SerialFieldMixin:
+    db_returning = True
+
+    def __init__(self, *args, **kwargs):
+        kwargs["blank"] = True
+        super().__init__(*args, **kwargs)
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        kwargs.pop("blank")
+        return name, path, args, kwargs
+
+    def check(self, **kwargs):
+        return [
+            *super().check(**kwargs),
+            *self._check_not_null(),
+            *self._check_default(),
+            *self._check_database_default(),
+        ]
+
+    def _check_not_null(self):
+        if self.null:
+            return [
+                checks.Error(
+                    "SerialFields do not accept null values.",
+                    obj=self,
+                    id="fields.E013",
+                ),
+            ]
+        else:
+            return []
+
+    def _check_default(self):
+        if self.default is NOT_PROVIDED:
+            return []
+        return [
+            checks.Error(
+                "SerialFields do not accept default values.",
+                obj=self,
+                id="fields.E014",
+            ),
+        ]
+
+    def _check_database_default(self):
+        if self.db_default is NOT_PROVIDED:
+            return []
+        return [
+            checks.Error(
+                "SerialFields do not accept default database values.",
```

Comment:
```suggestion
                "SerialFields do not accept default database values (db_default).",
```
This aligns better to `fields.E011`

Reply:
yes, I suppose it's clearer this way, thanks!